lib/edit-site-page.php

```diff
@@ -28,12 +28,7 @@ class="edit-site"
  * @return bool True for Site Editor pages, false otherwise.
  */
 function gutenberg_is_edit_site_page( $page ) {
-	$allowed_pages = array(
-		'gutenberg_page_gutenberg-edit-site',
-		'toplevel_page_gutenberg-edit-site',
-	);
-
-	return in_array( $page, $allowed_pages, true );
+	return 'toplevel_page_gutenberg-edit-site' == $page;
```

Comment:
Should we use `===` here?

Also, I wonder if this function is still necessary.  I guess it does help with readability either way, but most of our original reasons for adding it are gone.

Reply:
> Should we use === here?

Yes, I think we should. :)

> Also, I wonder if this function is still necessary. I guess it does help with readability either way, but most of our original reasons for adding it are gone.

I think it's still helpful compared to hardcoding this string all over the place. We might need to change it again in the future, and this way we don't have to update it in multiple other files each time. Also there might be other conditions that we want to add in the future on top of the existing one.